mx.truffleruby/mx_truffleruby.py

```diff
@@ -19,7 +19,7 @@
     import mx_truffleruby_benchmark
 
 _suite = mx.suite('truffleruby')
-rubyDists = ['TRUFFLERUBY', 'TRUFFLERUBY-TEST']
+root = _suite.dir
```

Comment:
I don't think it would require anything - it would just look to see if the suite is there anyway, and if it is and `SULONG_HOME` isn't set then set it for you. It would make it work automatically in the GraalVM repo, but if you don't have Sulong as another suite in your workspace then it wouldn't do anything.